Rdmp.Core/CommandExecution/AtomicCommands/ExecuteCommandSet.cs

```diff
@@ -48,7 +52,23 @@ public class ExecuteCommandSet:BasicCommandExecution
             _property = _setOn.GetType().GetProperty(property);
 
             if(_property == null)
+            {
                 SetImpossible($"Unknown Property '{property}'");
+
+                //suggest similar sounding properties
+                var suggestions =
+                    _setOn.GetType().GetProperties().Where(c => CultureInfo.CurrentCulture.CompareInfo.IndexOf(c.Name,property, CompareOptions.IgnoreCase) >= 0).ToArray();
+
+                StringBuilder msg = new StringBuilder($"Unknown Property '{property}'");
+
+                if (suggestions.Any())
```

Comment:
Newline needed between the 'unknown property' and 'did you mean' lines? Also this seems to build one big string then append it all to the stringbuilder, instead of appending each piece to the builder, which should technically be a little more efficient?

Reply:
Good catch.